doc/TRY.md

```diff
@@ -102,6 +116,7 @@ git checkout 6a6983e15e4d3c8c03937f8bee040de350ce722f
 make simulate
 ```
 
+This step might take a few minutes at the first launch of the Specter emulator.
```

Comment:
A few dozens of seconds maybe, but a few minutes?? Anyway, it doesn't hurt so much to have it.

Reply:
Yes, about 5 min for both Maria and I if i recall well

Reply:
Ok